CONTRIBUTING.md

```diff
@@ -1,98 +1,152 @@
-choose from a number of people to review your code.
-If you are modifying an existing feature try and find out who the
-original author of the code was and asking them to review your
-changes.
-If you are working on a new feature you can choose the most appropriate
-person.
-When you have decided on someone assign the Gitlab issue to them of
-leave an `@username` comment asking them to review.
+Next we need to apply our Django applications migrations to the database to do this
+let's first bring the database container back up, but this time run it in the
+background.
 
-### Merging your feature
+```sh
+docker-compose up -d postgres
+```
 
-Your code should not be merged into `develop` until it has been given
-the :thumbsup: by another developer (the merge request should literally
-have a thumbs up).
+After a few seconds you should see the postgres container running when you run
 
-When you're ready to merge, click the `Merge` button in the Gitlab
-merge request page, (also ensure the `Delete source branch` checkbox is
-ticked).
+```sh
+docker-compose ps
+```
 
-### Clean up your local branchs
+Next we'll using the `docker-compose run` command to run the Django `migrate` command.
 
-Feature branches are short lived, now that your feature is in develop
-delete your local feaure branch.
+```sh
+docker-compose run django python manage.py migrate
+```
+
+You will now be able to create a super user. This will ask for a username, email
+and password.
+
+```sh
+docker-compose run django python manage.py createsuperuser
+```
+
+Now that the superuser has been created you can login to the Django Admin site to
+ensure everything is running correctly.
+First start the application containers.
 
 ```sh
-git branch -d feature/BODP-1234-timetable-list-widget
+docker-compose up
 ```
 
-## Releasing a version of BODS
+Navigate to [admin.localhost:8000/admin](http://admin.localhost:8000/admin) an
+login with username and password you have previously created.
+
+There are a few Celery tasks that need to be run to populate the application
+with data.
+
+Navigate to the [Periodic Tasks](http://admin.localhost:8000/admin/django_celery_beat/periodictask/)
+in the Django Admin.
+
+Select the checkbox for the following period tasks
+
+- `run_naptan_etl`
+- `update_bods_xsd_zip_files`
+
+Select `Run selected tasks` from the dropdown and click `Go`.
+
+These tasks will populate the tables storing NaPTAN data and also fetch the schema
+files required to perform schema checks.
+
+## OTC Dataset
+
+BODS utilises the Office of Traffic Commissioners Bus Service dataset for
+Compliance and Monitoring functionality.
+To synchronise BODS with this data there are two Celery Tasks that need to be run.
+
+  1. task_get_all_otc_data
+  2. task_refresh_otc_data
+
+The first task should be run once on setup to ensure the BODS OTC tables are
+populated and the second task should be run periodically to ensure data stays
+up to date.
```

Comment:
Worth a warning to say that the first task takes several hours to complete. (I think it took about 6 hours when I ran it).